plugins/premium/mentions.md

```diff
@@ -271,6 +270,7 @@ tinymce.init({
       userRequest[id] = fetch('/user?id=' + id);
     }
     userRequest[id].then(function(userDetail) {
+      // Create a custom select element
```

Comment:
Should this be `hover element`? As that's what it's referred to in the original sentence.